### PR TITLE
Add descriptions of the GitHub labels to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hawkrives**
I added a bunch of label descriptions

Thoughts?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+Would you like to contribute? Great! Have a look at [React Native's "getting started" guide](http://facebook.github.io/react-native/docs/getting-started.html) to help you get your computer set up, and make a pull request! (If you feel like you should have write access to the repo, please make an issue and we can discuss it.)
 
-If you are looking to get a start with the project, issues tagged with [`good first change`](https://github.com/StoDevX/AAO-React-Native/issues?q=is%3Aissue+label%3A%22good+first+change%22+is%3Aopen) are a great place to start. These issues are ones that the team has identified as simple, small changes that won't get too deep, and will be easy to approve for merging.
+If you are looking to get started with the project, issues tagged with [`good first change`](https://github.com/StoDevX/AAO-React-Native/issues?q=is%3Aissue+label%3A%22good+first+change%22+is%3Aopen) are a great place to start. These issues are ones that the team has identified as simple, small changes that won't get too deep, and will be easy to approve for merging.
```
Comment (drewvolz) [inline CONTRIBUTING.md]: Looks like the label should be `good first issue`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+- <kbd>data/*</kbd> – Regarding our various data sources: bus routes, BonApp menus, building hours, and everything else
+- <kbd>dep/*</kbd> – Used when reporting bugs to blame either a JS or native dependency
+- <kbd>discussion</kbd> – Highlights issues that are in "discussion"
+- <kbd>good first issue</kbd> – Highlights anything we've identified as a good way to get your feet wet
+- <kbd>greenkeeper</kbd> – Identifies PRs submitted by Greenkeeper, our automated npm dependency updater
+- <kbd>help wanted</kbd> – We don't really use it. I guess it's used on PRs when someone needs help? But we usually just mention someone by name instead
+- <kbd>platform/*</kbd> – Used to show that a problem/feature request only applies to one platform or the other
+- <kbd>pr/*</kbd> – These originate from before Github had their Review system, but it's still helpful for these to stick around, because they let the author of the PR say "okay, this is ready for someone to review" or "this is broken right now"
+- <kbd>release-note-worthy</kbd> – Not entirely sure why this exists. I think it's so that we can use it to filter down the PRs that were merged when we write up the release notes?
```
Comment (drewvolz) [inline CONTRIBUTING.md]: Yep

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+    - <kbd>type/shipping hold</kbd> – highlights issues that are considered to block the next release
+    - <kbd>type/tracking</kbd> – high-level issues used to keep track of another set of issues
+- <kbd>view/*</kbd> – Used to scope an issue to a particular component in the app
+- <kbd>wontfix</kbd> – We won't fix anything with this label
+
+
+### Keep It Runnning
 We use a continuous-integration (CI) system to make sure that the project still works as we change things. Any submissions you make will be validated by [TravisCI](https://travis-ci.org/StoDevX/AAO-React-Native/builds).
 
 We use a set of tools to enforce code style and find common bugs: [ESLint][eslint], [Flow][flow], [Jest][jest], and [Prettier][prettier].
```
Comment (drewvolz) [inline CONTRIBUTING.md:37]: Why is it that Prettier's link does not go to https://prettier.io?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,7 +1,37 @@
...
+    - <kbd>type/documentation</kbd> – used for PRs/issues about documentation
+    - <kbd>type/enhancement</kbd> – used to create a list of things that we want to add/change about the app, but that aren't bugs
+    - <kbd>type/refactoring</kbd> – identifies issues/PRs about refactoring the app
+    - <kbd>type/shipping hold</kbd> – highlights issues that are considered to block the next release
+    - <kbd>type/tracking</kbd> – high-level issues used to keep track of another set of issues
+- <kbd>view/*</kbd> – Used to scope an issue to a particular component in the app
+- <kbd>wontfix</kbd> – We won't fix anything with this label
+
+
+### Keep It Runnning
```
Comment (drewvolz) [inline CONTRIBUTING.md:34]: Can you please change the `npm run foo` commands to `yarn` for the entire document?

Comment (hawkrives) [reply CONTRIBUTING.md:34]: I wasn't planning on changing any of the commands…

File: CONTRIBUTING.md (hunk 3 above)
Comment (hawkrives) [reply CONTRIBUTING.md:37]: Because this document predates that website

File: CONTRIBUTING.md (hunk 4 above)
Comment (drewvolz) [reply CONTRIBUTING.md:34]: If we're using yarn, why not suggest it?

File: CONTRIBUTING.md (hunk 3 above)
Comment (drewvolz) [reply CONTRIBUTING.md:37]: Any way to update that?

File: CONTRIBUTING.md (hunk 1 above)
Comment (drewvolz) [reply CONTRIBUTING.md]: So this link needs to change

File: CONTRIBUTING.md (hunk 3 above)
Comment (hawkrives) [reply CONTRIBUTING.md:37]: You are welcome to push to this branch

File: CONTRIBUTING.md (hunk 4 above)
Comment (hawkrives) [reply CONTRIBUTING.md:34]: Because this doc predates us using yarn, and I was only adding the label descriptions in this PR.

**Comment by drewvolz** [comment]
If we’re “updating” contributing, I figured we’d make all the updates we noticed.

**Comment by hawkrives** [comment]
I mis-named the PR, apologies. It should be titled "Add descriptions of the labels to CONTRIBUTING"

**Review by drewvolz — APPROVED** [review]

**Comment by drewvolz** [comment]
Good to go :shipit:

**Pull request merged**